skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -576,13 +576,14 @@ void MpActor::DropItem(const uint32_t baseId, const Inventory::Entry& entry)
   // TODO: Take count into account
   int count = entry.count;
   RemoveItems({ entry });
-  PapyrusObjectReference papyrusObjectReference;
-  auto baseForm = VarValue(std::make_shared<EspmGameObject>(
-    GetParent()->GetEspm().GetBrowser().LookupById(baseId)));
-  auto aCount = VarValue(count);
-  auto aForcePersist = VarValue(false);
-  auto aInitiallyDisabled = VarValue(false);
-  (void)papyrusObjectReference.PlaceAtMe(
-    this->ToVarValue(),
-    { baseForm, aCount, aForcePersist, aInitiallyDisabled });
+  // spawning items will be implemented later...
+  // PapyrusObjectReference papyrusObjectReference;
+  // auto baseForm = VarValue(std::make_shared<EspmGameObject>(
+  //   GetParent()->GetEspm().GetBrowser().LookupById(baseId)));
+  // auto aCount = VarValue(count);
+  // auto aForcePersist = VarValue(false);
+  // auto aInitiallyDisabled = VarValue(false);
+  // (void)papyrusObjectReference.PlaceAtMe(
+  //   this->ToVarValue(),
+  //   { baseForm, aCount, aForcePersist, aInitiallyDisabled });
```

Comment:
Let's either remove it entirely or make it multiline comment (`/* ... */`) to minimize diff